src/Sandbox.cs

```diff
@@ -37,21 +37,22 @@ namespace Gauge.Dotnet
     public class Sandbox : ISandbox
     {
         private readonly IAssemblyLoader _assemblyLoader;
+        private readonly IActivatorWrapper activatorWrapper;
+        private readonly Type instanceManagerType;
+        private readonly IReflectionWrapper reflectionWrapper;
 
         private object _classInstanceManager;
 
         private IHookRegistry _hookRegistry;
-        private readonly IActivatorWrapper activatorWrapper;
-        private readonly IReflectionWrapper reflectionWrapper;
-        private readonly Type instanceManagerType;
 
-        public Sandbox(IAssemblyLoader assemblyLoader, IHookRegistry hookRegistry, IActivatorWrapper activatorWrapper, IReflectionWrapper typeWrapper)
+        public Sandbox(IAssemblyLoader assemblyLoader, IHookRegistry hookRegistry, IActivatorWrapper activatorWrapper,
```

Comment:
Probably worth refactoring this as well. Since there is no sandboxing anymore, I don't think we need `Sandbox` class. Inline/rename this. I leave it up to you to decide if this is worth a separate PR

Reply:
Yeah. That's what I want to do. I didn't want to bloat this PR with everything.